zerver/tornadoviews.py

```diff
@@ -24,11 +29,13 @@
 
 @internal_notify_view
 def notify(request):
+    # type: (HttpRequest) -> HttpResponse
     process_notification(ujson.loads(request.POST['data']))
     return json_success()
 
 @has_request_variables
 def cleanup_event_queue(request, user_profile, queue_id=REQ()):
+    # type: (HttpRequest, UserProfile, Optional[str]) -> HttpResponse
```

Comment:
I think `queue_id` isn't optional here -- though there may not be a corresponding client_descriptor